docs/assemblies/assembly-overview.adoc

```diff
@@ -15,6 +15,21 @@ It takes on the responsibility of receiving, filtering, and forwarding messages.
 
 An API provides a convenient means for implementing custom logic within the proxy.
 
+[id='assembly-overview-compatability{context}']
+== Compatibility
+
+[id='assembly-overview-compatability-api{context}']
+=== APIs
+Kroxylicious follows https://https://semver.org/#semantic-versioning-200[Semantic Versioning] rules. While we are still in the `0.y.z` phase we still take API compatability very seriously, we aim to provide at least two minor releases between deprecation and the removal of that deprecated item.
+
+We also consider our configuration model to be a "public API", and thus it will follow the same Semantic Versioning and deprecation rules.
+
+Kubernetes Custom Resources are obviously considered to be a public API, and we will make every effort to evolve them inline with Kubernetes best practice.
```

Comment:
this might be our first mention of anything operator related, it might be a bit odd for users to see a mention of CRs without any futher information about an Operator.

Reply:
Fair, it just seemed natural to include them given our current focus on building them... What does anyone else think?

Reply:
Might be worth a line in the _Kroxylicious overview_ section before going into compatibility as there's an API sentence there too.